tc_hivemind_backend/tests/unit/test_mongo_singleton.py

```
@@ -0,0 +1,108 @@
+            with self.assertRaises(Exception) as context:
+                MongoSingleton()
+            self.assertEqual(str(context.exception), "This class is a singleton!")
+
+    def test_successful_connection(self):
+        """Test successful MongoDB connection logging"""
+        with patch("tc_hivemind_backend.db.mongo.MongoClient") as mock_client:
+            # Mock successful connection
+            mock_client_instance = mock_client.return_value
+            mock_client_instance.server_info.return_value = {"version": "4.4.0"}
+
+            instance = MongoSingleton.get_instance()
```

Ruff

Local variable `instance` is assigned to but never used

Remove assignment to unused variable `instance`

(F841)